app/frontend/js/CreateModal.js

```diff
@@ -0,0 +1,12 @@
+import { Modal } from "bootstrap"
+
+
+const createModal = document.getElementById('create-modal');
+
+if(createModal) {
+  const alert = document.getElementsByClassName('is-invalid');
+
+  if(alert[0]) {
+    new Modal(document.getElementById('create-modal')).show();
+  }
+}
```

Comment:
This is all looking very cryptic. Maybe it needs comments on what it's for? Why do we check for the `is-invalid` etc...

Reply:
Yeah sure thing! It's just to open the modal when the page refreshes and there is an error :)